backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/AbstractSortedDocValues.java

```diff
@@ -14,7 +14,10 @@
  * {@link DocIdSetIterator} APIs. This impl is safe to use for sorting and
  * aggregations, which only use {@link #advanceExact(int)} and
  * {@link #ordValue()}.
- */
+ * <p>
+ * Copied and adapted from {@code PUT THE ORIGINAL CLASS NAME HERE}
```

Comment:
I have removed "AbstractSortedDocValues.java ". I use a different solution. Everything is "DoubleMultiValuesSource" and "LongMultiValuesSource". This can sort and aggregate.